src/packer.pkr.hcl

```
@@ -136,19 +143,56 @@ build {
   }
 
   provisioner "windows-restart" {
-    # Wait a maximum of 30 minutes for Windows to restart.
-    # The build will fail if the restart process takes longer than 30 minutes.
-    restart_timeout = "30m"
+    # Wait a maximum of 5 minutes for Windows to restart.
+    # The build will fail if the restart process takes longer than 5 minutes.
```

Same idea where I think it reads better as a continuous comment.

```suggestion
    # Wait a maximum of 5 minutes for Windows to restart. The build will fail
    # if the restart process takes longer than 5 minutes.
```